implementations/rust/ockam/ockam_node/src/storage/database/database_configuration.rs

```diff
@@ -30,6 +30,8 @@ pub enum DatabaseConfiguration {
     Postgres {
         /// Connection string of the form postgres://[{user}:{password}@]{host}:{port}/{database_name}
         connection_string: String,
+        /// Path to a SQLite database that needs to be migrated to the Postgres database.
+        legacy_sqlite_path: Option<PathBuf>,
```

Comment:
Do we plan to remove the related logic to this field once we have completed the migrations wherever it's needed?

Reply:
Potentially yes, but it's not impossible that we start some independent nodes at some stage and then want to move them to being managed nodes. So I don't know how long this code should stay in.